en/docs/setup/migrating-to-5110.md

```diff
@@ -138,28 +138,23 @@ Once all the above prerequisites have been met, follow the instructions given be
 
         Make sure to point the internal keystore to the keystore that is copied from the previous WSO2 Identity Server version. The primary keystore can be pointed to a keystore with a certificate that has a strong RSA key.
 
-4.  To migrate the super tenant, copy the content (except the webapps folder) from `<OLD_IS_HOME>/repository/deployment/server` directory to the `<NEW_IS_HOME>/repository/deployment/server` directory. 
-
-    If you have customized the web apps in the previous WSO2 Identity Server version, those changes should be manually applied to the relevant files with careful inspection. If you have created tenants in the previous WSO2 Identity Server
-    version that contain resources, copy the content from `<OLD_IS_HOME>/repository/tenants` directory,
-    to the `<NEW_IS_HOME>/repository/tenants` directory.
-    
-5.  If you have created secondary user stores in the previous WSO2 IS
-    version, copy the content in the
-    `           <OLD_IS_HOME>/repository/deployment/server/userstores          `
-    directory to the
-    `           <NEW_IS_HOME>/repository/deployment/server/userstores          `
-    directory.
+4.  If you have created tenants in the previous WSO2 Identity Server version, copy the content from the `<OLD_IS_HOME>/repository/tenants` folder to the `<NEW_IS_HOME>/repository/tenants` folder.
```

Comment:
According to [1], only secondary user stores should be migrated for other tenants. If that is true, we can specifically say that. wdyt?

[1] https://github.com/wso2-enterprise/wso2-iam-internal/issues/115

Reply:
Since the `tenant` folder covers the user stores as well, we will stick with this wording.